java/semantickernel-api/src/main/java/com/microsoft/semantickernel/hooks/KernelHook.java

```diff
@@ -6,7 +6,19 @@
 import java.util.function.Function;
 import java.util.function.Predicate;
 
-public interface KernelHook<T extends KernelHookEvent> extends Predicate<KernelHookEvent>, Function<T, T> {
+public interface KernelHook<T extends KernelHookEvent> extends Predicate<KernelHookEvent>,
+    Function<T, T> {
+
+    /**
+     * The priority of the hook. The default priority is 1000. The priority is used to determine the
+     * order in which hooks are executed, higher priorities are executed first. No ordering is
```

Comment:
"in which hooks that accept the same event" ? 

Also, 1000? What about a range of [1..100]? Or at least positive values. 

Should priority be numerical order, lower values have more priority 1="I'm first in line"